app/crud.py

```diff
@@ -0,0 +1,50 @@
+from sqlalchemy.orm import Session
+
+from app import schemas, models
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):

```

Comment:
Add type hint for the returned value. Check other functions and add type hint for the returned value too.